### PR TITLE
Get context for event

### PR DESCRIPTION
**Pull request by berggren**
### Event context query

This PR makes it possible to query from a event and get +- N seconds of context around that event.

---

Ref issue: #60 
Reviewer: @onager

### CODE REVIEW

File: timesketch/api/v1/resources.py [hunk 1]
```diff
@@ -298,62 +287,69 @@ def get(self, sketch_id):
...
+        if form.validate_on_submit():
+            query_filter = form.filter.data
+            sketch_indices = [
+                t.searchindex.index_name for t in sketch.timelines]
+            indices = query_filter.get(u'indices', sketch_indices)
 
-        # Make sure we have a query string or star filter
-        if not args.get(u'q') and not query_filter.get(u'star'):
-            abort(HTTP_STATUS_CODE_BAD_REQUEST)
+            # Make sure that the indices in the filter is part of the sketch
```
Comment (Onager) [inline timesketch/api/v1/resources.py]: "filter are part"

File: timesketch/ui/views/home.py [hunk 2]
```diff
@@ -46,6 +47,8 @@ def home():
         Sketch.Status.parent).order_by(Sketch.updated_at.desc())
     query_filter = request.args.get(u'filter', u'')
     query = request.args.get(u'q', u'')
+    # Only render upload button if it is configures.
```
Comment (Onager) [inline timesketch/ui/views/home.py]: configured

File: timesketch/api/v1/resources.py (hunk 1 above)
Comment (berggren) [inline timesketch/api/v1/resources.py]: done

File: timesketch/ui/views/home.py (hunk 2 above)
Comment (berggren) [inline timesketch/ui/views/home.py]: done.

**Pull request merged**